base/release_strategy.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: managed
 spec:
   pipeline: m5-release-pipeline
-#  bundle: quay.io/hacbs-release/demo:m5-alpine
+  bundle: quay.io/hacbs-release/m5-release-pipeline:main
   policy: m5-policy
   params:
     - name: extraConfigGitUrl
       value: https://github.com/davidmogar/strategy-configs.git
     - name: extraConfigPath
       value: m5.yaml
-    - name: extraConfigRevision # optional field, default is main
```

Comment:
Why remove this?

Reply:
My bad. I was testing stuff locally and I probably send the wrong patch. Fixing it.